crates/uv/tests/sync.rs

```diff
@@ -2328,3 +2328,44 @@ fn transitive_dev() -> Result<()> {
 
     Ok(())
 }
+
+#[test]
+/// Check warning message for https://github.com/astral-sh/uv/issues/6998.
+fn sync_scripts_without_build_system() -> Result<()> {
```

Comment:
Do you think it's worth adding a test case for `package = false` with a build system defined?

Reply:
Added, together with a test and a TODO for `uv run`.